diff --git a/data/autosuspend.hwdb b/data/autosuspend.hwdb
--- a/data/autosuspend.hwdb
+++ b/data/autosuspend.hwdb
@@ -323,7 +323,11 @@ usb:v061Ap0110*
 usb:v061Ap0110*
  ID_AUTOSUSPEND=1
  ID_PERSIST=0
-
+# Supported by libfprint driver verimark
+usb:v047Dp00F2*
+ ID_AUTOSUSPEND=1
+ ID_PERSIST=0
+
 # Supported by libfprint driver vfs0050
 usb:v138Ap0050*
  ID_AUTOSUSPEND=1
@@ -357,6 +361,5 @@ usb:v138Ap0091*
 # Known unsupported devices
 usb:v0A5Cp5802*
-usb:v047Dp00F2*
 usb:v047Dp8055*
 usb:v04E8p730B*
 usb:v04F3p036B*
diff --git a/libfprint/fprint-list-udev-hwdb.c b/libfprint/fprint-list-udev-hwdb.c
--- a/libfprint/fprint-list-udev-hwdb.c
+++ b/libfprint/fprint-list-udev-hwdb.c
@@ -30,7 +30,6 @@ static const FpIdEntry allowlist_id_table[] = {
    *   gio cat https://gitlab.freedesktop.org/libfprint/wiki/-/wikis/Unsupported-Devices.md | sed -n 's!|.*\([0-9a-fA-F]\{4\}\):\([0-9a-fA-F]\{4\}\).*|.*!  { .vid = 0x\1, .pid = 0x\2 },!p'
    */
   { .vid = 0x0a5c, .pid = 0x5802 },
-  { .vid = 0x047d, .pid = 0x00f2 },
   { .vid = 0x047d, .pid = 0x8055 },
   { .vid = 0x04e8, .pid = 0x730b },
   { .vid = 0x04f3, .pid = 0x036b },
diff --git a/tests/meson.build b/tests/meson.build
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -335,7 +335,7 @@ appstreamcli = find_program('appstreamcli', required: false)
 if appstreamcli.found()
     test('metainfo-validate',
         appstreamcli,
-        args: ['validate', metainfo_generator],
+        args: ['validate', '--no-net', metainfo_generator],
         depends: metainfo_generator,
         suite: ['data', 'no-valgrind'],
     )
